src/ProductConstructionService/ProductConstructionService.BarViz/Pages/PullRequests.razor

```diff
@@ -4,71 +4,92 @@
 @using Microsoft.DotNet.ProductConstructionService.Client.Models
 @using Microsoft.FluentUI.AspNetCore.Components.Extensions
 @using System.Linq.Expressions
+@using ProductConstructionService.BarViz.Code.Helpers
 @using ProductConstructionService.BarViz.Components
 @inject IProductConstructionServiceApi PcsApi
 
+<style>
```

Comment:
Shouldn't we keep the CSS in one place?

Reply:
This has to go with this component though

Reply:
It's actually an advice from the docs